_includes/specials/taproot/en/08-signature-adaptors.md

```diff
@@ -0,0 +1,111 @@
+eventually revealed, the donor could sabotage Alice and Bob's contract
+by giving them an incorrect hash.
+
+### Adaptor magic
+
+[Signature adaptors][topic adaptor signatures], also commonly called
+*adaptor signatures* and *one-time verifiably encrypted signatures*, are
+a solution to these problems---and to many other problems actually faced
+today in production systems built on Bitcoin.  Although
+usable with Bitcoin's existing ECDSA signature scheme, it's much easier
+to use adaptors privately and costlessly in combination with the
+[BIP340][] implementation of [schnorr signatures][topic schnorr
+signatures] for taproot.  Let's see how our example above changes if we
+use adaptors.
+
+As before, the donor prepares a 1,000 BTC transaction.  They sign in
+almost the normal way, with the one difference being that they
+essentially generate their nonce in two parts: a true random nonce that
+they will forever keep secret, and their favorite number---which they'll
+initially keep secret but which is safe for other people to discover.
+The donor generates a fully valid signature using both of these values,
+adding them together as if they were a single nonce.
+
+BIP340 signature commitments use the nonce in two forms: a numeric
+representation (called a *scalar*), which normally only the signer
+knows, and as a *point* on the Elliptic Curve (EC), which is published
+to enable verification.
+
+The donor takes the commitment part of their valid signature and
+subtracts out the hidden scalar.  This makes the signature incomplete
+(and thus invalid) but allows the donor to share the (invalid) signature
+commitment, the (valid) point for the complete nonce, and the (valid)
+point for the hidden number.  Together these three pieces of information
+are a *signature adaptor*.
+
+Using a slight variant on the BIP340 signature verification algorithm,
+anyone can verify that signature adaptor would provide a valid signature
+if the hidden scalar was simply added back in to the (currently invalid)
+signature commitment.  This is possible to verify even without knowing
+what that hidden number is.  In short, it is now possible for users to
+trustlessly begin making guesses about the value of hidden scalar,
+secure in the knowledge that a correct guess will allow them to get the
+signature and send the transaction.
+
+Like everyone else who received the donor's signature adaptor, Alice and
+Bob now have a copy of the EC point for the hidden number.  Also like
+everyone else, they don't know the actual scalar.  But, if you recall,
+all the donor did to turn their valid signature into an invalid
+signature is subtract out the hidden number from their signature
+commitment while continuing to have the signature commit to the point
+of the hidden number.  Alice can just as easily create an invalid
+signature by not committing to the scalar she doesn't know but still
+committing to the EC point she does know.  This allows Alice to create a
```

Comment:
I find "not committing to the scalar she doesn't know but still committing to the EC point she does know" just a little difficult to understand. What does it mean that Alice's signature is _not committing to the scalar she doesn't know_?